src/OpenTelemetry.SemanticConventions/scripts/semantic-convetion/generate.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Do we usually have a `.ps1` version for each `.sh` script or specially these days with `wsl` it doesn't matter?
Definitely easier to maintain only one script.

Reply:
It's there now :). Tested it produces the same result as the other.